data/redirects.yml

```diff
@@ -595,3 +595,7 @@
   url: "/enclave/"
 - loc: "/training/"
   url: "/"
+- loc: "/assets/aptible-enclave-division-of-responsibilities.pdf/"
```

Comment:
@henryhund - I think you need to remove this slash at the end or it won't ever match